todoapp/users/serializers.py

```
@@ -1,10 +1,48 @@
+
+        if exclude is not None:
+            for field_name in exclude:
+                self.fields.pop(field_name)
+
+
+class UserTodoStatSerializer(UserSerializer):
+    completed_count = serializers.IntegerField()
+    pending_count = serializers.IntegerField()
+
+    class Meta(UserSerializer.Meta):
+        fields = UserSerializer.Meta.fields + ["completed_count", "pending_count"]
```

Ruff (0.12.2)

Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)